packages/@tinacms/cli/src/cmds/init/index.ts

```diff
@@ -139,7 +134,7 @@ export async function tinaSetup(ctx: any, next: () => void, options) {
   const tinaBlogPagePathFile = p.join(tinaBlogPagePath, '[filename].tsx')
   if (!fs.pathExistsSync(tinaBlogPagePathFile)) {
     fs.mkdirpSync(tinaBlogPagePath)
-    fs.writeFileSync(tinaBlogPagePathFile, nextPostPage({ wrapper }))
+    fs.writeFileSync(tinaBlogPagePathFile, nextPostPage())
```

Comment:
Should we still pass wrapper here? That way we can render the edit state in the generated file if they choose not to use a _app.js ?

Reply:
You mean dump it out into `/components/tina-wrapper`? I'm not sure we should give them anything else to worry about. We'd like them to just put it in `_app.js` I think

Reply:
Previously if the user did not want to update there _app.js the CLI would wrap /demo/blog/[filename].tsx with the edit provider and Tina Provider so that edit would still work on that page (it would not work on the rest of their site). We dont have to worry about maintaining that behaviour for this PR I can add it back latter if needed.

Reply:
I think it would be ideal to merge and test this before tomorrow. Probably better to merge and release it before we worry about other features.

Reply:
> Previously if the user did not want to update there _app.js the CLI would wrap /demo/blog/[filename].tsx

Ok I see, instead of adding it to a separate file would it be better to just wrap the default export in `/demo/blog/[filename].tsx`? That way it's all housed in one file.